packages/fx-core/src/plugins/resource/identity/index.ts

```diff
@@ -111,9 +111,9 @@ export class IdentityPlugin implements Plugin {
       return err(error);
     }
 
-    ctx.config.set(Constants.identityName, this.config.identityName);
-    ctx.config.set(Constants.identityId, this.config.identityId);
-    ctx.config.set(Constants.identity, this.config.identity);
+    ctx.config.set(Constants.identityName, this.config.identity);
+    ctx.config.set(Constants.identityClientId, this.config.clientId);
```

Comment:
Do we need to update the migration logic to change the name of related properties in `env.default.json`?